conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/ApiCirceJsonSchema.scala

```diff
@@ -1,36 +1,14 @@
 package tech.cryptonomic.conseil.api.routes.platform.data
 
-import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
-import akka.http.scaladsl.server.Directives.complete
-import akka.http.scaladsl.server.Route
-import akka.util.ByteString
 import cats.Functor
-import endpoints.akkahttp.server.{Endpoints => AkkaEndpoints, JsonSchemaEntities => AkkaJsonSchemaEntities}
-import endpoints.algebra.Documentation
+import endpoints.akkahttp.server.circe.{JsonSchemaEntities => AkkaCirceJsonSchema}
 import io.circe._
 import io.circe.syntax._
-import tech.cryptonomic.conseil.api.routes.platform.data.CsvConversions._
 import tech.cryptonomic.conseil.common.generic.chain.DataTypes
 import tech.cryptonomic.conseil.common.generic.chain.DataTypes._
-import tech.cryptonomic.conseil.common.util.Conversion.Syntax._
 
 /** Trait with methods for converting from data types to Json */
-trait ApiDataHelpers extends AkkaEndpoints with AkkaJsonSchemaEntities with ApiDataJsonSchemas {
-
-  /** Function validating request for the query endpoint */
-  def defaultValidated[A](
-      response: A => Route,
-      invalidDocs: Documentation
-  ): Either[List[QueryValidationError], A] => Route = {
-    case Left(errors) =>
-      complete(StatusCodes.BadRequest -> s"Errors: \n${errors.mkString("\n")}")
-    case Right(QueryResponseWithOutput(queryResponse, OutputType.csv)) =>
-      complete(HttpEntity.Strict(ContentTypes.`text/csv(UTF-8)`, ByteString(queryResponse.convertTo[String])))
-    case Right(QueryResponseWithOutput(queryResponse, OutputType.sql)) =>
-      complete(HttpEntity.Strict(ContentTypes.`text/plain(UTF-8)`, ByteString(queryResponse.head("sql").get.toString)))
-    case Right(success) =>
-      response(success)
-  }
+trait ApiCirceJsonSchema extends AkkaCirceJsonSchema with ApiDataJsonSchemas {
```

Comment:
Where has the validation actually gone? Which module does it now?

Reply:
`ApiValidation` does that.